src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -3311,7 +3311,7 @@ public sealed class DnsNameProperty
         private readonly List<DnsNameRepresentation> _dnsList = new();
         private readonly IdnMapping idnMapping = new();
 
-        private const string distinguishedNamePrefix = "CN=";
+        private const string CommonNameOid = "2.5.4.3";
```

Comment:
If we want to follow strong API this should be Oid type.

Reply:
Thanks @iSazonov, very nice pickup. Fixed in f45d90608570a298b11486afde29daf1e0a6a24b